theme/templates/resource-landing-page/funding-agency.html

```diff
@@ -110,6 +110,21 @@ <h4 class="modal-title" id="delete-funding-agency">Delete Funding Agency</h4>
                 </div>
 
                 <div class="modal-body">
+                    <div v-if="unmatchedFunders.length > 0" class="alert alert-info alert-dismissible" role="alert">
+                        <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+                          <span aria-hidden="true">&times;</span>
+                        </button>
+                        <div class="flex">
+                          <i class="glyphicon glyphicon-info-sign" style="margin-right: 20px;"></i>
+                          <em style="padding-right:20px;">
+                            This resource contains funders (listed below) that do not exist in the <a href="https://www.crossref.org/services/funder-registry" target="_blank">Open Funder Registry</a>:
+                            <ul>
+                              <li v-for="funder in unmatchedFunders">${funder}</li>
+                            </ul>
+                            We recommend updating the funders to conform to the <a href="https://www.crossref.org/services/funder-registry" target="_blank">Open Funder Registry</a> to ensure consistency and ease of reporting.
+                          </em>
+                        </div>
+                      </div>
```

Comment:
why are you including this code block within the `DELETE FUNDING AGENCY MODAL`? This seems like a pretty strange place for it. Can you explain why you are moving it?

Reply:
@MSDrao I still think this is pretty funky: showing the notification within the delete modal:
<img width="1365" alt="image" src="https://github.com/user-attachments/assets/5a7bf66a-704b-4291-a69d-3438f815a410">

https://github.com/user-attachments/assets/08fc177d-0a1d-46b8-a429-02977f6ac933

I think I would still recommend keeping this notification where it was before

Reply:
@martinseul what is your opinion? ☝️ (see video to see what we are talking about)

Reply:
I'd agree with Devin these are separate task and should not be conflated